Add note for section placement in snap graphics (#320)
MirServer/ubuntu-frame: docs/how-to/use-snap-graphics.md

       target: $SNAP/gpu-2404
       default-provider: mesa-2404
   ```

> Note: this section must be inserted before 'parts:' and 'apps:'.

Reviewer: That's not true, YAML doesn't care about order?

The only thing is that it needs to be placed as a top level key?

Author: For most snapcrafters (newbies) 'plugs:' is a section that we use/put into 'apps:'. But for this one, as I understand (and tested), it must not be placed into 'apps:' nor 'parts:', otherwise the command 'snapcraft pack' does fail.

Reviewer 2: Yes, there are two "plugs" keys in the snapcraft grammer: "apps.plugs" and "plugs".

But this note is actively misleading and doesn't address the potential for confusion you describe.

Reviewer: Yes, it needs to be placed as a top-level key, but not "before".

https://documentation.ubuntu.com/snapcraft/9.0/reference/snapcraft-yaml/#content-plug-keys

```suggestion

   Note: this must be inserted as a {ref}`top-level plugs: entry <snapcraft:project.plugs>`.
```
